doc/source/ray-contribute/profiling.rst

```diff
@@ -1,7 +1,7 @@
 .. _ray-core-internal-profiling:
 
-Profiling (internal)
-====================
+Profiling
```

Comment:
Same comment as for "Debugging" Wondering if we should use a more explicit title.

Reply:
ack!